lib/absinthe/relay/node/parse_ids.ex

```diff
@@ -229,7 +229,17 @@ defmodule Absinthe.Relay.Node.ParseIDs do
   @spec find_schema_root!(Absinthe.Type.Field.t(), Absinthe.Resolution.t()) ::
           {{Absinthe.Type.Field.t() | Absinthe.Type.Argument.t(), String.t()},
            (String.t() -> String.t())}
-  defp find_schema_root!(field, resolution) do
+  defp find_schema_root!(
+         %{
+           __private__: [
+             absinthe_relay: [
+               payload: {:fill, _},
+               input: {:fill, _}
```

Comment:
I think this works. It's a bit fragile, but as long as we get a test case in here then if Absinthe.Relay's implementation changes then the test will fail and we can use a different mechanism.

Reply:
Do you have a suggestion on a less frail way to implement this? I was trying to think of a better way, but this is what I came up with since __parse_ids_root: is set on the resolution level and not the field level.

Reply:
Yeah I think as long as we get some test cases in we'll be OK. The fragility is all about Absinthe.Relay internals, there isn't anything the end user can do to cause issues.

Reply:
@benwilson512 I added some test cases. I wasn't sure what is acceptable in your repo and what isn't, so let me know if this works and/or if you'd prefer it tested a different way.

Reply:
Resolving this as outdated.